kivy/uix/gesturesurface.py

```diff
@@ -238,9 +238,13 @@ class GestureSurface(FloatLayout):
             Color used to draw the gesture, in RGB. This option does not
             have an effect if :attr:`use_random_color` is True.
 
-            :attr:`draw_timeout` is a
+            :attr:`color` is a
             :class:`~kivy.properties.ListProperty` and defaults to
```

Comment:
```suggestion
            :class:`~kivy.properties.ColorProperty` and defaults to
```

Reply:
The existing approach is the same as in previous pull request. Like this for example: https://github.com/kivy/kivy/blob/master/kivy/uix/actionbar.py#L214, and that was approved by matham.

Reply:
I think I overlooked that by only looking at the diff 😬 sorry

Reply:
mistakes have been made :P